### PR TITLE
Allow translation of structure fields in blocks

### PR DESCRIPTION
**Pull request by tillschander**

### CODE REVIEW

File: src/panel/mixins/translation.js [hunk 1]
```diff
@@ -16,6 +16,36 @@ export default {
           if (!Object.keys(translatableBlocks).includes(block.type)) continue;
 
           for (const blockFieldKey of Object.keys(block.content)) {
+
+            // Handle structures
+            if (
+              isObject(translatableBlocks[block.type]) &&
```
Comment (tillschander) [inline src/panel/mixins/translation.js:22]: Currently "isObject" also returns true for arrays. I don't know if this is intended. But here i want to specifically check for objects only.

Comment (johannschopplich) [reply src/panel/mixins/translation.js:22]: You can use `isObject(obj) && !Array.isArray(obj)`.

**Comment by johannschopplich** [comment]
Nice one! Thanks for the PR. Will look into it later. Great work. 👏

File: src/panel/mixins/translation.js (hunk 1 above)
Comment (tillschander) [reply src/panel/mixins/translation.js:22]: Yeah that's what I ended up doing. Just thought I'd make you aware of this potential pitfall. But it seems like you already are aware of it :)

**Comment by tillschander** [comment]
Even better, great!

**Comment by tillschander** [comment]
And thanks for your quick update :)

**Pull request closed without merging**